deserialize/type_checks.py

```diff
@@ -73,9 +87,24 @@ def list_content_type(type_value):
     return type_value.__args__[0]
 
 
+def tuple_content_types(type_value):
+    """Strip the Tuple wrapper from a type.
+
+    e.g. Tuple[int, str] -> [int, str]
+    """
+
+    if not is_tuple(type_value):
+        raise TypeError(f"{type_value} is not a Tuple type")
+
+    return type_value.__args__
```

Comment:
How does this handle `Tuple[str, ...]` (which gives you a tuple of arbitrary length, all of type `str`)?